coreos/gce-create-cluster.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
well, i think it can be helpfull because it is not straightforward to figure out how to add a disk.
It is not required to use it.